chain/network/src/types.rs

```diff
@@ -280,7 +288,8 @@ impl From<AnnounceAccountRoute> for network_proto::AnnounceAccountRoute {
             peer_id: announce_account_route.peer_id.into(),
             hash: announce_account_route.hash.into(),
             signature: announce_account_route.signature.into(),
-            ..Default::default()
+            cached_size: Default::default(),
```

Comment:
`..Default::default()`

Reply:
It's easier to track down the missing fields in the Into converters this way.